Add JavaScript Network Layer

Kikobeats opened the pull request:
I don't know if the JavaScript section is the more correct... what do you think?

sindresorhus commented on readme.md:
```diff
@@ -64,6 +64,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 
 - [JavaScript](https://github.com/sorrycc/awesome-javascript)
 	- [Promises](https://github.com/wbinnssmith/awesome-promises)
+	- [Network Layer](https://github.com/Kikobeats/awesome-network-js/blob/master/README.md#awesome-javascript-network--)
```
Link to the repo, not the readme file.

sindresorhus commented:
Would you mind reading the [contributing guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md) :)

sindresorhus commented:
> Your awesome list should have the awesome badge, a license, and contribution guidelines. Adding a code of conduct is also strongly recommended. - https://github.com/sindresorhus/awesome/blob/master/create-list.md

sindresorhus commented:
And the merge conflict needs to be fixed.

Kikobeats commented:
Added license → https://github.com/Kikobeats/awesome-network-js/blob/master/LICENSE.md
Added contributing → https://github.com/Kikobeats/awesome-network-js/blob/master/CONTRIBUTING.md
